cli/src/cli.mjs

```diff
@@ -425,6 +428,29 @@ if ( parsed.help ) {
     );
   }
 
+  if ( parsed.sessionOutputPath ) {
+    const sessionOutputPath = parsed.sessionOutputPath;
+    msg(`Set session output path to: ` + sessionOutputPath);
+    await axios.post(url(host, port, '/api/v1/sessionoutputpath'), { path : sessionOutputPath })
+    .then(function (response) {
```

Comment:
Same comment as above; the use of await and .then and .catch seems unusual. A try-catch around the await seems more appropriate if we want to catch errors.